.github/workflows/lint.yml

```diff
@@ -11,7 +11,7 @@ permissions:
 
 jobs:
     lint:
-        runs-on: ubuntu-20.04
```

Comment:
This is deprecated and removed as of 2025-04-15, so the action is failing on this PR.